core/templates/pages/blog-dashboard-page/services/blog-dashboard-page.service.ts

```diff
@@ -33,7 +33,7 @@ import { BlogPostData } from 'domain/blog/blog-post.model';
 export class BlogDashboardPageService {
   private _blogPostId: string = '';
   private _authorPictureUrl: string = '';
-  private _blogPostData: null | BlogPostData = null;
+  private _blogPostData!: BlogPostData;
```

Comment:
From where we know that `_blogPostData` won't be `null`?

Reply:
Done